scripts/us_census/geojsons_low_res/generate_mcf.py

```diff
@@ -44,17 +45,19 @@ def __init__(self):
         self.eps = 0.01
 
     def download_data(self):
-        self.downloader.download_data(place=self.PLACE_TO_IMPORT)
+        self.downloader.download_data(place=self.PLACE_TO_IMPORT,
+                                      level=self.LEVELS_DOWN)
 
     def generate_simple_geojsons(self, show=False, eps=None):
+        original_pts = 0
+        compressed_pts = 0
         for geoid, coords in self.downloader.iter_subareas():
-
             # This an admitedly hacky way of getting rid of bogus geographies
             # that are, for some mysterious reason, included as part of the US
             # states query (with geoIds like geoId/7000).
             # TODO(jeffreyoldham): (potentially) find a cleaner way to do this.
-            if int(geoid.split('/')[1]) > 100:
-                continue
+            # if int(geoid.split('/')[1]) > 100:
+            #     continue
```

Comment:
Why are these commented out?

Reply:
This filter removed undesired state geoIds. Since in this case we're importing counties, this constraint doesn't make sense (all county geoIds are above 100).

Reply:
Let's protect with it the appropriate conditional for states rather than commenting it out.